inspector/src/components/actionTabs/tabs/propertyGrids/animations/timeline.tsx

```diff
@@ -159,27 +246,133 @@ export class Timeline extends React.Component<
     this.setState({ activeKeyframe: null });
   }
 
+  private active: string = '';

```

Comment:
Could we replace with _active

Reply:
yes, makes sense... all privates should have underscore... _ :)